README.md

```diff
@@ -10,6 +10,23 @@ The development version of this package supports Django 5.0.x. To install it:
 
 `pip install git+https://github.com/mongodb-labs/django-mongodb`
 
+### Via templates
+
+To create a new Django project that uses MongoDB, you can use the templates in this repository. For example:
+
+```console
+$ django-admin startproject --template=project_template mysite
```

Comment:
I'm not sure if we should include these templates in this repo. Normally, users will be pip installing this package, which means they would either have to reference `--template=/path/to/site-packages/django_mongodb/project_template` or install [via a URL](https://github.com/jpadilla/django-project-template?tab=readme-ov-file#how-to-install). It looks like you modified the source distribution not to include the templates, so it's unclear how you're expecting this command to work. Probably installing via URL makes the most sense, which means we could include them in this repo and exclude them from the source, or a separate repo could work just as well. We should have separate templates for each version of Django, which could just be branches.

Reply:
> I'm not sure if we should include these templates in this repo. Normally, users will be pip installing this package, which means they would either have to reference `--template=/path/to/site-packages/django_mongodb/project_template` or install [via a URL](https://github.com/jpadilla/django-project-template?tab=readme-ov-file#how-to-install). It looks like you modified the source distribution not to include the templates, so it's unclear how you're expecting this command to work. 

I'm running these local but I'm not sure either … now would be a good time to decide. I think I'm leaning slightly towards closing this PR and creating templates in a separate repository, particularly if we're going to be providing branches for each major Django release.

> Probably installing via URL makes the most sense, which means we could include them in this repo and exclude them from the source, or a separate repo could work just as well. We should have separate templates for each version of Django, which could just be branches.

Agree! And the [example](https://github.com/jpadilla/django-project-template) you provided is nice. What do you think @Jibola ?